.changeset/modern-islands-cheer.md

```diff
@@ -0,0 +1,7 @@
+---
+"@cambly/syntax-core": major
```

Comment:
Do we have any other breaking changes lined up that we'd like to include in this bump?